scss/new-test-runner.scss

```diff
@@ -0,0 +1,17 @@
+@import 'inc/bootstrap';
+@import 'inc/colors';
+// @import "inc/loading-bar";
+// @import "inc/action-bars";
+// @import 'inc/section-container';
```

Comment:
they are into TAO right ? we need to sort this out

Reply:
Yes, all 3 are in `tao-core` and I do not know, how should I load them.